grpc/src/client/load_balancing/mod.rs

```diff
@@ -151,13 +153,13 @@ pub(crate) trait LbPolicyBuilder: Send + Sync {
 /// LB policies are responsible for creating connections (modeled as
 /// Subchannels) and producing Picker instances for picking connections for
 /// RPCs.
-pub trait LbPolicy: Send {
+pub trait LbPolicy: Send + Debug {
     /// Called by the channel when the name resolver produces a new set of
     /// resolved addresses or a new service config.
     fn resolver_update(
         &mut self,
         update: ResolverUpdate,
-        config: Option<&LbConfig>,
+        config: Option<LbConfig>,
```

Comment:
We can avoid clones by sending references. For example, pickfirst doesn't seem to be storing the LB config presently. The ChildManager is still cloning the config to pass an owned object to all its children. With a reference, LB policies can choose to clone if they need an owned object.

Is supporting references complicating the implementation?

Reply:
> For example, pickfirst doesn't seem to be storing the LB config presently

Regarding this, note that PF only uses the lb config for its behavior _during_ `resolver_update` which I believe is much less common than configuring the behavior of its ongoing operation.

Reply:
Thinking more about this, it seems most instances of `LbConfig` will actually be references -- they will come out of parts of the service config.  So perhaps this way is best...